crowdsourcing/serializers/user.py

```diff
@@ -100,6 +114,8 @@ def validate_username(self, value):
         return value
 
     def create(self, **kwargs):
+        #import pdb
+        #pdb.set_trace()
```

Comment:
@aditimithal we shouldn't check in commented out code. You can check in comments that describe what you're doing but this makes the code cluttered. :)